src/components/common/ParentNavTopBar.js

```diff
@@ -40,6 +40,14 @@ const ParentNavTopBar = props => {
         </Title>
       </a>
       <div style={{ display: 'flex', alignItems: 'center' }}>
+        <Link
+          to={{
+            pathname: '/parent/faq',
+          }}
+        >
+          <span className="welcome-back-msg">FAQ</span>
+        </Link>
+        <span className="welcome-back-msg">Contact Us |</span>
         <span className="welcome-back-msg">
           {props.parent && `Welcome back, ${props.parent.name}`}
         </span>
```

Comment:
added links to Mosharrafs FAQ page so that it could be demoed at the stakeholder meeting